display-guild.lic

```diff
@@ -0,0 +1,91 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#display-guild
+=end
+
+no_pause_all
+no_kill_all
+
+arg_definitions = [
+    [
+	{ name: 'debug', regex: /debug/i, optional: true, description: 'Very verbose, may cause lich to hang for several seconds if using large contact files' }
+	]
+]
+args = parse_args(arg_definitions)
+$debug_mode_cc = args.debug
+
+Contacts = YAML.load_file("./scripts/data/base-contacts.yaml")
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+echo "---Base Contacts---" if $debug_mode_cc
+echo Contacts.inspect if $debug_mode_cc
+echo Contacts.class if $debug_mode_cc
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?("./scripts/profiles/#{checkname}-contacts.yaml")
+	echo "Personal contact file found - now importing." if $debug_mode_cc
+	
+	Personal_Contacts = YAML.load_file("./scripts/profiles/#{checkname}-contacts.yaml")

```

Comment:
I agree that it should not be part of your normal YAML. The dependency manager I linked you to (as well as bankbot) has an example of saving a YAML file in the lich root directory so it can be accessed by multiple characters.